Application/Business/UserServices/UserServices.cs

```diff
@@ -25,39 +29,31 @@ public async Task<int> Count()
     }
     public async Task Create(UserEntity user) {
         var userList = List();
-        var check = false;
         foreach(var User in userList.Result)
         {
             if(User.Email == user.Email)
             {
-                check = true;
-                break;
+                throw new ConflictException("Email exists");
```

Comment:
I don't understand why you return this kind of exception. Let return bad request when this is client error

Reply:
```
public async Task Create(UserEntity user) {
        var userList = List();
        foreach(var User in userList.Result)
        {
            if(User.Email == user.Email)
            {
                throw new BadRequestException("Email exists"); // Here
            }
        }
        await _userRepo.CreateAsync(user);
    }
```